app-emulation/lxd/lxd-2.16.ebuild

```diff
@@ -40,12 +40,13 @@ EGO_VENDOR=(
 	"gopkg.in/yaml.v2 eb3733d160e74a9c7e442f435eb3bea458e1d19f github.com/go-yaml/yaml"
 )
 
-ARCHIVE_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+ARCHIVE_URI="https://${EGO_PN}/archive/${P}.tar.gz -> ${P}.tar.gz"
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64"
 
-PLOCALES="de el fr it ja nl ru sr sv tr"
+# Portage complains about zh_Hans missing, but repoman doesn't like it when it's there.
+PLOCALES="de el fr it ja nl ru sr sv tr zh"
```

Comment:
`LINGUAS` USE flags are deprecated. You should really install those unconditionally, or have some implicit `LINGUAS` handling (if you really insist). However, I won't delay this fix because of that.